### PR TITLE
chapter 5 created

### PR DESCRIPTION
**Pull request by agustinaGallo97**
## :link: Jira Tracker reference:
* [AP - Chapter 5] Your Second Activity - https://androidtraining.atlassian.net/browse/AT-42
## Description:
Chapter 5 created

## :camera_flash: UI preview
ezgif com-video-to-gif(4)

### CODE REVIEW

File: GeoQuizChallenge/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -12,16 +12,13 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         >
-
+        <activity android:name=".views.CheatActivity"></activity>
```
Comment (FerrariBruno) [inline GeoQuizChallenge/app/src/main/AndroidManifest.xml]:
```suggestion
        <activity android:name=".views.CheatActivity" />
```

File: GeoQuizChallenge/app/src/main/java/com/bignerdranch/android/geoquiz/views/CheatActivity.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+public class CheatActivity extends AppCompatActivity {
+  private static final String EXTRA_ANSWER_IS_TRUE = "com.bignerdranch.android.geoquiz.answer_is_true";
+  private static final String EXTRA_ANSWER_SHOWN = "com.bignerdranch.android.geoquiz.answer_is_shown";
+
+  private boolean answerIsTrue;
+  private TextView answerTextView;
+  private Button showAnswerButton;
+
+  public static Intent newIntent(Context packageContext, boolean answerIsTrue) {
```
Comment (FerrariBruno) [inline GeoQuizChallenge/app/src/main/java/com/bignerdranch/android/geoquiz/views/CheatActivity.java]: nullability

File: GeoQuizChallenge/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -12,16 +12,13 @@
...
-
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-            
         </activity>
-
     </application>
 
-</manifest>
+</manifest>
```
Comment (FerrariBruno) [inline GeoQuizChallenge/app/src/main/AndroidManifest.xml]: nl

File: GeoQuizChallenge/app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -12,16 +12,13 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         >
-
+        <activity android:name=".views.CheatActivity" />
```
Comment (mirland) [inline GeoQuizChallenge/app/src/main/AndroidManifest.xml:16]:
```suggestion

        <activity android:name=".views.CheatActivity" />
```

File: GeoQuizChallenge/app/src/main/java/com/bignerdranch/android/geoquiz/views/CheatActivity.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.bignerdranch.android.geoquiz.R;
+
+public class CheatActivity extends AppCompatActivity {
```
Comment (mirland) [inline GeoQuizChallenge/app/src/main/java/com/bignerdranch/android/geoquiz/views/CheatActivity.java:14]: nullability in all methods

File: GeoQuizChallenge/app/src/main/java/com/bignerdranch/android/geoquiz/views/CheatActivity.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+    showAnswerButton = findViewById(R.id.showAnswerButton);
+    showAnswerButton.setOnClickListener(v -> {
+      answerTextView.setText(answerIsTrue ? R.string.button_true : R.string.button_false);
+      setAnswerShownResult(true);
+    });
+  }
+
+  private void setAnswerShownResult(boolean isAnswerShown) {
+    Intent data = new Intent();
```
Comment (mirland) [inline GeoQuizChallenge/app/src/main/java/com/bignerdranch/android/geoquiz/views/CheatActivity.java]: chain it

File: GeoQuizChallenge/app/src/main/res/layout/activity_cheat.xml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```
Comment (mirland) [inline GeoQuizChallenge/app/src/main/res/layout/activity_cheat.xml:2]: you can use a frameLayout

**Review by mirland — APPROVED** [review]

File: GeoQuizChallenge/app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -13,4 +13,8 @@
     <string name="correct">Correct</string>
     <string name="incorrect">Incorrect</string>
     <string name="percentage_score">Percentage score is: </string>
+    <string name="warning_text">Are you sure you want to do this? </string>
+    <string name="show_answer_button">Show answer</string>
+    <string name="cheat_button">Cheat</string>
```
Comment (f7deleon) [inline GeoQuizChallenge/app/src/main/res/values/strings.xml]: maybe name it just cheat?

**Review by FerrariBruno — APPROVED** [review]

**Pull request merged**